doc/lua_api.txt

```diff
@@ -2104,11 +2104,19 @@ Elements
 
 * Show an inventory image of registered item/node
 
-### `bgcolor[<color>;<fullscreen>]`
-
-* Sets background color of formspec as `ColorString`
-* If `true`, a fullscreen background is drawn and the color is ignored
-  (does not affect the size of the formspec)
+### `bgcolor[<bgcolor>;<fullscreen>;<fbgcolor>]`
+
+* Sets background color of formspec.
+* `bgcolor` and `fbgcolor` are `ColorString`s, they define the color of the
```

Comment:
`fbgcolor (optional)`

Also `fullscreen` should be mentioned as optional argument.